internal/flags.go

```diff
@@ -27,6 +27,7 @@ type InputFlags struct {
 	summary                  bool
 	breakingOnly             bool
 	checkBreaking            bool
+	changelog                bool
```

Comment:
started with a -changelog flag, but planning to create a separate PR for moving to sub-commands: help, version, diff, breaking, changelog

Reply:
Please update usage section in README.md